### ISSUE TITLE
SearchTransactionsApi not parsing the approvalCode field

### ISSUE THREAD

**Issue by grigorecretu88** [CyberSource/cybersource-rest-client-java#46]
Hi,

I'm currently trying to integrate the rest client into our project and I'm facing the following issue.
When trying to retrieve a transaction using the `SearchTransactionsApi#createSearchWithHttpInfo` method, the returned `TssV2TransactionsPost201Response.embedded.transactionSummaries.processorInformation` object doesn't contain the `approvalCode` field. When using an equivalent HTTP request to retrieve the same data, I can see that the received JSON **does cotain**  the `approvalCode` field.

Received JSON structure:

```
...........
"transactionSummaries": [
            {
                .........
                "processorInformation": {
                    ..........
                    "approvalCode": "888888"
                }
                .........
            }
]
...........

```

Since the data for the `approvalCode` is already provided by the REST API, would it be possible to add the following missing field mapping into the `TssV2TransactionsPost201ResponseEmbeddedProcessorInformation` class ?

```
  @SerializedName("approvalCode")
  private String approvalCode = null;
```

Kind regards!

**Comment by snavinch** [comment]
Thank you for bringing this to our attention. We will inform the respective API team and work towards bringing this to a closure

**Comment by gnongsie** [comment]
This issue has been resolved in v0.0.20.

Kindly update to the latest version and try again.

Do reach out to us if you face any other issues.
